client/src/components/application/NonlinearStepper/style.js

```diff
@@ -38,6 +38,10 @@ const GeneralInfoStyles = makeStyles((theme) => ({
     marginTop: theme.spacing(1),
     marginBottom: theme.spacing(1),
   },
+  textArea: {
+    outlineColor: '#ed6d23',
+    marginLeft: '1rem',
+  },
 }));
```

Comment:
please try to fix the style as you can , to be shown for us at least , I can't accept it with this shape, Noting appear well